docker-entrypoint.sh

```diff
@@ -36,6 +41,7 @@ case "${DEPLOYMENT_MODE}" in
     if [ "$LOAD_PROD_DATA" == "on" ]; then
       # Option for migration_test.sh to source directly from production
       python ./joplin/manage.py loaddata ./joplin/db/system-generated/tmp_production.datadump.json
+      sanitize_revision_data
```

Comment:
Why only on Load_Prod_Data?

Is there an assumption that if you run with this flag locally once it will fix the problem moving forward?

Reply:
Load_Prod_Data is only run during migration-test, when your seeding data is initially created. So if you sanitize that data, then the datadump thats used as a future seeding.datadump.json will also be sanitized.